### PR TITLE
Cleanup run config UI (#258).

### PR DESCRIPTION
**Pull request by pq**
* removes unused “program arguments” field
* sets default proposed name to project name (rather than “Unnamed”)
* renames “Dart file” label to “Dart entry point”

Fixes: #258 

/cc @alexander-doroshko @devoncarew

### CODE REVIEW

**Comment by devoncarew** [comment]
Screenshot?

**Comment by devoncarew** [comment]
lgtm!

File: src/io/flutter/run/FlutterConfigurationEditorForm.form [hunk 1]
```diff
@@ -28,28 +28,12 @@
         <properties>
           <inheritsPopupMenu value="true"/>
           <labelFor value="17e7a"/>
-          <text value="Dart &amp;file:"/>
+          <text value="Dart &amp;entry point:"/>
```
Comment (pq) [inline src/io/flutter/run/FlutterConfigurationEditorForm.form]: :)

**Comment by pq** [comment]
screen shot 2016-11-03 at 2 11 45 pm

**Pull request merged**